src/main/resources/templates/admin/cohortView.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE HTML>
+<html xmlns:th="http://www.thymeleaf.org">
+<head th:replace="~{/admin/header :: head}"/>
+<body>
+
+<span th:replace="~{/admin/header :: top}"/>
+
+<a href="/admin/">Home</a>
+ -
+<a href="/admin/cohorts">Cohorts</a>
+
+<th:block th:if="${canUpdateCohort}">
```

Comment:
If you decide to make this available in read-only form to TF Expert and Read Only users, this conditional will probably need to turn into something like a series of conditionals on the submit buttons to grey them out.